config/locales/client.en.yml

```diff
@@ -2040,7 +2040,7 @@ en:
       refresh_page: "Refresh page"
 
     summary:
-      in_progress: "Summarizing topic using AI..."
+      in_progress: "Summarizing topic using AI"
```

Comment:
btw. is the animated ellipsis positioned correctly in the case of RTL? 😃

Reply:
> btw. is the animated ellipsis positioned correctly in the case of RTL? 😃

<img width="256" alt="image" src="https://github.com/discourse/discourse/assets/30537603/1f97b84c-35af-467f-b938-bdfb09916f61">

Looks correct. This sentence is still in english because it hasn't had a translation made for it yet.